WBooks/BookLibrary/BookLibraryController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  BookTableController.swift
+//  WBooks
+//
+//  Created by Federico Agustin Diaz on 25/07/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class BookLibraryController: UIViewController {
+    private let _view: TableView = TableView.loadFromNib()!
+    @available(*, unavailable, message: "use init(viewModel:) instead")
+    required public init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    var libraryItems: [Book] = []
+    
+    @available(*, unavailable, message: "use init(viewModel:) instead")
+    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        fatalError("init(nibName:bundle:) has not been implemented")
+    }
+    
+    init() {
+        super.init(nibName: .none, bundle: .none)
+    }
+    
+    override public func loadView() {
+        view = _view
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        configureTableView()
+        
+        libraryItems = BookInfo.sharedInstance.getAllBooks()
+        
+    }
+    
+    private func configureTableView() {
+        _view.table.delegate = self
+        _view.table.dataSource = self
+        _view.table.contentInset = UIEdgeInsets(top: 10, left: 0, bottom: 0, right: 0)
+        let nib = UINib.init(nibName: "BookCell", bundle: nil)
+        _view.table.register(nib, forCellReuseIdentifier: "BookCell")
```

Comment:
This is OK. But in WolmoCore we have something that makes it easier (https://github.com/Wolox/wolmo-core-ios/blob/master/WolmoCore/Extensions/UIKit/UITableView.swift) `register(cell:)`